api/bean/UserRequest.go

```diff
@@ -31,7 +31,7 @@ type UserRole struct {
 
 type UserInfo struct {
 	Id            int32        `json:"id" validate:"number,not-system-admin-userid"`
-	EmailId       string       `json:"emailId" validate:"required,not-system-admin-user"`
+	EmailId       string       `json:"email_id" validate:"required,not-system-admin-user"`
```

Comment:
we can have emailId as well and can start using it in newer versions, email_id is anyway incorrect convention and can be deprecated in future

Reply:
it has an impact on user creation and updation as well, have to sent this urgently. can be done in future releases

Reply:
add TODO comment so that it does not gets missed later